packages/expo-image-picker/ios/EXImagePicker/EXImagePicker.m

```diff
@@ -336,9 +341,16 @@ - (BOOL)tryCopyImage:(NSDictionary * _Nonnull)info path:(NSString *)toPath {
   return false;
 }
 
-- (void)handleVideoWithInfo:(NSDictionary * _Nonnull)info saveAt:(NSString *)directory updateResponse:(NSMutableDictionary *)response
+- (void)handleVideoWithInfo:(NSDictionary * _Nonnull)info
+                     saveAt:(NSString *)directory
+             updateResponse:(NSMutableDictionary *)response
+          completionHandler:(void (^)(void))completionHandler
 {
-  NSURL *videoURL = info[UIImagePickerControllerMediaURL];
+  NSURL *videoURL = info[UIImagePickerControllerMediaURL] ? info[UIImagePickerControllerMediaURL] : info[UIImagePickerControllerReferenceURL];
+  if (videoURL == nil) {
+    self.reject(@"E_COULDNT_OPEN_FILE", @"Couldn't open video", nil);
```

Comment:
I don't really like when we're defining resolve/reject callbacks as an instance member of the module. It is considered as a bad practice since it can easily break when someone call the method twice without awaiting. Don't change that now, I just wanted you to keep that in mind in the future 😉